packages/core/test/predictableExec.test.ts

```diff
@@ -8,7 +8,9 @@ describe('predictableExec', () => {
 
     const machine = createMachine({
       initial: 'a',
-      context: {},
+      context: {
+        assigned: false
+      },
```

Comment:
This isn't used by the test anyhow
```suggestion
      context: {},
```